### PR TITLE
1.1 Test scenario Game of all misses

### LINKED ISSUE #5: 0.2 add main-ut.cpp

### PR DESCRIPTION
**Pull request by majkel84**
new file added
CMakeList udpated

### CODE REVIEW

**Review by Vicx95 — DISMISSED** [review]

**Comment by arafalski** [comment]
Why do you want to add second main for tests? We've already added `gtest_main` in `CMakeLists.txt`
Line 63:
```cmake
target_link_libraries(${PROJECT_NAME}-ut PUBLIC ${PROJECT_NAME}-lib gtest_main)
```

File: bowling/main-ut.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+        testing::InitGoogleTest(&argc, argv);
+        auto code = RUN_ALL_TESTS();
+        return code;
+    }
+    catch (...)
+    {
+        return 0;
+    }
+    return 0;
+}
```
Comment (arafalski) [inline bowling/main-ut.cpp]: Redundant file

**Review by arafalski — CHANGES_REQUESTED** [review]

File: bowling/main-ut.cpp (hunk 1 above)
Comment (majkel84) [reply bowling/main-ut.cpp]: done

**Review by arafalski — APPROVED** [review]

**Review by arafalski — CHANGES_REQUESTED** [review]
Rebase/merge with anraf1001:homework
Solve conflicts

**Pull request merged**